ec2/spark_ec2.py

```diff
@@ -878,44 +888,56 @@ def wait_for_cluster_state(conn, opts, cluster_instances, cluster_state):
 # Get number of local disks available for a given EC2 instance type.
 def get_num_disks(instance_type):
     # Source: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html
-    # Last Updated: 2014-06-20
+    # Last Updated: 2015-05-08
     # For easy maintainability, please keep this manually-inputted dictionary sorted by key.
     disks_by_instance = {
         "c1.medium":   1,
         "c1.xlarge":   4,
+        "c3.large":    2,
+        "c3.xlarge":   2,
         "c3.2xlarge":  2,
         "c3.4xlarge":  2,
         "c3.8xlarge":  2,
-        "c3.large":    2,
-        "c3.xlarge":   2,
-        "cc1.4xlarge": 2,
```

Comment:
ah so I was going off of:

http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html

Its absense may be an error, or intentional from AWS.  I've seen:

EC2 is steering support away from cc1.4xlarge instances and toward
cc2.8xlarge instances. The cc2.8xlarge instance supports additional
features such as VPC and availability in all regions. When creating
instances in the RightScale Dashboard, use cc2.8xlarge instances instead of
cc1.4xlarge.



On Fri, May 8, 2015 at 3:36 PM, Shivaram Venkataraman <
notifications@github.com> wrote:

> In ec2/spark_ec2.py
> <https://github.com/apache/spark/pull/6014#discussion_r29969797>:
>
> >          "c3.2xlarge":  2,
> >          "c3.4xlarge":  2,
> >          "c3.8xlarge":  2,
> > -        "c3.large":    2,
> > -        "c3.xlarge":   2,
> > -        "cc1.4xlarge": 2,
>
> Looks like cc1.4xlarge has been removed - Is that intentional ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/6014/files#r29969797>.
>